recipes/allegro5/all/conanfile.py

```diff
@@ -19,7 +19,7 @@ class Allegro5Conan(ConanFile):
         "fPIC": True
     }
     
-    generators = "cmake", "cmake_find_package"
+    generators = "cmake", "cmake_find_package", "pkg_config"
```

Comment:
pkg_config in generators means that you have to add pkgconf in build requirements.